components/Card.js

```diff
@@ -21,56 +42,91 @@ function Card({ backgroundImageSource, backgroundImageAltText, children }) {
         </figure>
       )}
       <div
-        className={`card-body grid ${cardsBackgroundColor} text-${textColor} gap-y-3 auto-rows-card`}
+        className={`carousel-body flex flex-col p-6 shadow-lg justify-start items-center ${cardsBackgroundColor} text-${textColor} rounded-b-md`}
```

Comment:
I'm curious why you changed the class card-body to carousel-body

Reply:
now, I also don't remember 😆